apps/web/components/eventtype/settings/DisableAllEmailsSetting.tsx

```diff
@@ -0,0 +1,81 @@
+import type { TFunction } from "next-i18next";
+import { Trans } from "next-i18next";
+import { useState } from "react";
+
+import {
+  SettingsToggle,
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  InputField,
+  DialogClose,
+  Button,
+} from "@calcom/ui";
+
+interface DisableEmailsSettingProps {
+  checked: boolean;
+  onCheckedChange: (e: boolean) => void;
+  recipient: "attendees" | "hosts";
+  t: TFunction;
+}
+
+export const DisableAllEmailsSetting = ({
+  checked,
+  onCheckedChange,
+  recipient,
+  t,
+}: DisableEmailsSettingProps) => {
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const [confirmText, setConfirmText] = useState("");
+
+  const title =
+    recipient === "attendees" ? t("disable_all_emails_to_attendees") : t("disable_all_emails_to_hosts");
+
+  return (
+    <div>
+      <Dialog open={dialogOpen} onOpenChange={(e) => setDialogOpen(e)}>
+        <DialogContent title={title} Icon="circle-alert">
+          <p className="text-default text-sm">
+            <Trans i18nKey="disable_attendees_emails_description">
+              This will disable all emails to {{ recipient }}. This includes booking confirmations, requests,
+              reschedules and reschedule requests, and cancellation emails.
```

Comment:
We are missing the location change emails here (maybe some other email too)

Should we add something like 'and any other emails related to booking updates'